.github/workflows/develop.yml

```diff
@@ -47,26 +39,13 @@ jobs:
             with:
                 python-version: ${{ matrix.python-version }}
 
-        -   name: Install system dependencies
+        -   name: Install Python dependencies
             run: |
-                sudo apt update
-                sudo apt install postgresql-12
-
-        -   name: Install python dependencies
-            continue-on-error: true
-            id: install
-            run: |
-                pip install --upgrade setuptools
+                pip install --upgrade pip setuptools wheel
                 pip install git+https://github.com/aiidateam/aiida-core@develop#egg=aiida-core[atomic_tools]
                 pip install -e .[tests]
                 reentry scan
 
-        -   name: Run pytest
```

Comment:
Aren't you removing the actual step that runs the tests here?

Reply:
whoops, little too deletion-happy